github/Requester.py

```diff
@@ -454,7 +454,7 @@ def __setstate__(self, state: Dict[str, Any]) -> None:
         self.__custom_connections = deque()
 
     @staticmethod
-    # replace with str.removesuffix once support for Python 3.7 is dropped
+    # replace with str.removesuffix once support for Python 3.8 is dropped
     def remove_suffix(string: str, suffix: str) -> str:
```

Comment:
Can you replace the usage of `remove_suffix` with `str.removesuffix` and remove this method?

Reply:
It would break in Python 3.8. The comment was wrong, but it's fixed now.
`str.removesuffix` was added in 3.9, not 3.8.
See [PEP 616](https://peps.python.org/pep-0616/) as well as [Python Docs](https://docs.python.org/3.9/library/stdtypes.html#str.removesuffix):

Reply:
You are right, my bad!

Reply:
No worries, and thank you!